src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -0,0 +1,230 @@
+// @flow
```

Comment:
Not at this time I don't think, because it would require converting all the associated components to TS and didn't think that would be appropriate for the scope of this PR.